dashboard/db/migrate/20210722162824_update_backpack_indices.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateBackpackIndices < ActiveRecord::Migration[5.2]
+  def change
+    remove_index :backpacks, :user_id
```

Comment:
Up to you if you want to fix this but I _think_ this migration will fail if it needs to be rolled back for any reason. You can split this into an `up` and `down` function to mitigate that. It's pretty unlikely it'll be an issue though

Reply:
I tested rolling back just now and it worked without any changes!

Reply:
Yay awesome! Thanks for checking and sorry for the incorrect advice!

Reply:
no problem, definitely good to double check